### PR TITLE
add perl-text-unidecode

### PR DESCRIPTION
**Pull request by khaled196**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by khaled196** [comment]
@conda-forge/help-perl

the tool passed all the tests and it is ready to be reviewed

File: recipes/perl-text-unidecode/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+test:
+  imports:
+    - Text::Unidecode
+
+
+
+about:
+  home: http://metacpan.org/pod/Text-Unidecode
+  license: perl_5
```
Comment (bgruening) [inline recipes/perl-text-unidecode/meta.yaml:37]: can you maybe change the license to something like that https://github.com/conda-forge/perl-uri-feedstock/blob/main/recipe/meta.yaml#L41

Important is that its correct ;) and its a SPDX identifier. https://spdx.dev/

Comment (khaled196) [reply recipes/perl-text-unidecode/meta.yaml]: Hi Bjorn, 

I used the same licence as the example Perl package you sent me. I checked the package licence and they said they are using the same Perl licence, I searched for a Perl licence and found this

 Perl is an Open Source software, licensed under its Artistic License, or the GNU General Public License (GPL)
 
 As long as there is no different licence in the tool and they say they use a Perl licence I will use this form.

**Comment by khaled196** [comment]
Hi @bgruening, 

After adding licence testing Windows failed with the error code "can't find the package" so I skipped it.

**Review by bgruening — APPROVED** [review]
Thanks this looks good to me.

**Pull request merged**